packages/openai/src/openai-completion-language-model.test.ts

```diff
@@ -154,6 +189,24 @@ describe('doGenerate', () => {
     });
   });
 
+  it('should extract logprobs', async () => {
```

Comment:
Does this test cover streaming? Are we returning logbprobs when streaming?

Reply:
i'll double check

Reply:
good catch - I just ran `-u` on tests and didn't check too thoroughly if they matched the existing ones, i'll make sure to do that in the future 🙏